docs/design/ovs-pipeline.md

```diff
@@ -1350,30 +1374,31 @@ If you dump the flows of this table, you may see the following:
 ```
 
 Flows 1-2 match packets originating from local Pods and destined for the transport IP of remote Nodes, and then forward
-them to table [L2ForwardingCalc] to skip Egress SNAT. `ToGatewayRegMark` is loaded, indicating that the output port of
-the packets is the local Antrea gateway.
+them to table [L2ForwardingCalc] to bypass the Pod-to-Node traffic from Egress SNAT. `ToGatewayRegMark` is loaded,
```

Comment:
"to bypass the Pod-to-Node traffic from Egress SNAT" should be replaced with "to bypass Egress SNAT for Pod-to-Node traffic"